### ISSUE TITLE
Client ID and Secret

### ISSUE THREAD

**Issue by dgauthie** [tonesto7/nest-manager#519]
## Verify the following before opening an issue:

- [X ] You followed the Nest Developer documentation here? [Nest Developer Account Creation](https://cdn.rawgit.com/tonesto7/nest-manager/master/Documents/html/CreateID_Dev_Account.html)
- [X] That _OAuth_ is Enabled for the SmartApp under the IDE.
- [X] The SmartApps and *_ALL_* (5) Devices Handlers are using the latest code available.
- [X] You have reviewed the [NST Manager Wiki](http://thingsthataresmart.wiki/index.php?title=NST_Manager) for potential fixes.
- [ X] That ALL SmartApps and Device Handlers have been _Published for You_ in the IDE.

## About Your Setup:
* How many Thermostats?: 2
* How many Protects? 2
* How many Cameras? 0
* Using the Nest Presence Device? 0
* Using the Nest Weather Device? Yes
* iOS or Android?:IOS
* Mobile App Version:

## Expected Behavior
Nest Manager should start up

## Current Behavior
 "You are missing the Client ID and Secret"  "Create a Client ID and Secret"

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Context

Please include a copy of your log under the IDE if possible

**Comment by spencerthayer** [comment]
Same issue.

**Comment by DJDeleeted** [comment]
it took me a minute to find this as well but please follow instructions on this link
https://developers.nest.com/documentation/cloud/how-to-auth

**Comment by dgauthie** [comment]
The very kind people at Samsung Smartthings walked me thru the process approximately THREE WEEKS AGO when I was having difficulties!!!  If I had waited for your response, I would have pulled all of my hair out!!!

**Comment by slimshizn** [comment]
Have been using your smartapp for a while now. Thank you for your work and allowing me to delete the nest app all together. Recently ran into this issue when I noticed the commands weren't being sent to the thermostat anymore. Went into the smart app and it said the ID and secret were missing. Came on here and went to the developers website. Went through the steps and added the information to my smartthings IDE. Works good/better than it did.

**Comment by dgauthie** [comment]
Yes, I also had to go to the Nest site and become a Developer to get the ID and Secret when I added the Nest Smoke Detector to Smart Things!

**Comment by adam7950** [comment]
I followed all of the directions, created the developer account to obtain my Client Id and Secret added it to the Settings in Nest Manager, I know OAuth is enabled, I tripled check i copied and pasted properly and I still get Client ID adn Secret are both missing.  Only One wierd thing that I see is if I go back into settings the Client Secret appears on top then Client ID after that....any ideas that I can check?
